packages/lesmis/src/api/queries/useVitalsData.js

```diff
@@ -87,32 +87,30 @@ const useDistrictReport = entity =>
   );
 
 const useMultiSchoolReport = (entities, rootEntity) => {
-  const decendants = getDecendantCodesOfType(entities, rootEntity?.code, 'school');
+  const descendants = getDescendantCodesOfType(entities, rootEntity?.code, 'school');
   return useReport(
     rootEntity,
     'LESMIS_multi_school_vitals',
     {
-      data: {
+      params: {
         endDate: utcMoment().format(endDateFormat),
-        organisationUnitCodes: decendants.join(),
       },
     },
-    decendants.length > 0,
+    descendants.length > 0,
```

Comment:
While entity aggregation should be able to handle this (ie. it will just return no data if requested for an entity that has no school descendants), I figured it's still a good practice to keep these checks on the front-end to avoid redundant calls to report-server?

Reply:
I think this is fine for now. We probably want to remove it long term because processing these hierarchies on the front end causes a fair bit of slowdown (plus I think it's generally against best practices to do this much processing in the front end).

But I think this work is already above and beyond the scope required by you so I'm happy to leave that for team nz to deal with down the line.